.circleci/config.yml

```diff
@@ -98,6 +98,18 @@ jobs:
       - store_artifacts:
           path: ~/sg_logs
 
+  integration_test:
+    description: "running integration test on the code"
+    machine: true 
+    resource_class: deci-ai/research-standard
+    steps:
+        - checkout
+        - run:
+           name: integration test
+           #command: sudo docker run -it -e ENVIRONMENT_NAME=production -v ${PWD}:/SG -v /data:/data deci1/super-gradients:test python3 ./tests/integration_tests/pretrained_models_test.py
+           command: sudo docker run -it -e ENVIRONMENT_NAME=production -v ${PWD}:/SG -v /data:/data deci1/super-gradients:test python3 -c 'print("it works!")'
```

Comment:
@strelok899  whats is `deci1` repository?

Reply:
why is the environment production?

Reply:
> why is the environment production?

shai said